modules/gdscript/gdscript_editor.cpp

```diff
@@ -52,6 +52,7 @@
 #include "editor/editor_string_names.h"
 #include "editor/file_system/editor_file_system.h"
 #include "editor/settings/editor_settings.h"
+#include "scene/gui/code_edit.h"
```

Comment:
Is this dependency fine? Can `modules/gdscript` depend on code from `scene` without issue? Or does it need a compile define to access this safely